Lombiq.Tests.UI/Delegates/MultiSizeTest.cs

```diff
@@ -16,3 +17,18 @@ namespace Lombiq.Tests.UI.Delegates;
 
 /// <inheritdoc cref="MultiSizeTest"/>
 public delegate Task MultiSizeTestAsync(UITestContext context, bool isStandardSize);
+
+public static class MultiSizeTestExtensions
+{
+    [SuppressMessage(
+        "Minor Code Smell",
+        "S4261:Methods should be named according to their synchronicities",
+        Justification =
+            "This is a conversion method, and we want to make it explicit that the result has an async signature.")]
+    public static MultiSizeTestAsync ToAsync(this MultiSizeTest test) =>
```

Comment:
I think it's still misleading becuase you'd assume this method does something similar to `Task.Run()` instead of just returning `Task.CompletedTask`. So to me this hurts more than it helps. How about one of these mouthfuls?
- `ReturnCompletedTask`
- `WithCompletedTask`
- `ToMultiSizeTestAsyncWithCompletedTask`

Reply:
Maybe `.AsCompletedTask()`?

Reply:
That's fine too.

Reply:
I'm thinking that the `Completed` infix is a lot of implementation noise that shouldn't be there, don't you think? Considering just `.AsTask()` now.

Reply:
I think whether a task will actually start a thread or not is useful information, not noise.